### PR TITLE
[Docker] add docker

### PR DESCRIPTION
**Pull request by chan-stephane**
This pull request adds support for running the application in a Docker container. A new **Dockerfile** and **docker-compose.yml** has been added to run the application in Docker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ Currently, two official plugins are available:
 - [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
 - [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
 
+## Run in Docker
+Install Docker and docker compose and execute, the app run in http://localhost:8080/
```
Comment (puchka) [inline README.md:31]:
```suggestion
Install Docker and docker compose and execute the command below, the app run in http://localhost:8080/
```

**Review by puchka — CHANGES_REQUESTED** [review]
Nice PR @chan-stephane :100: 

I have just a little suggestion to improve the change in the README :wink:

File: README.md (hunk 1 above)
Comment (chan-stephane) [reply README.md:31]: Ok

**Review by puchka — APPROVED** [review]
I just applied the change! Good job! :+1:

**Pull request merged**